middlewares/logger.js

```diff
@@ -5,10 +5,15 @@ export default function logger(request, response, next) {
 
   const userAgent = request.headers['user-agent'];
 
+  const { origin } = request.headers;
+  const { referer } = request.headers;
```

Comment:
Poderia colocar na mesma linha

```suggestion
  const { origin, referer } = request.headers;
```